keras_retinanet/bin/evaluate.py

```diff
@@ -113,6 +113,7 @@ def main(args=None):
 
     # create the generator
     generator = create_generator(args)
+    custom_objects = custom_objects_importer(args.backbone)
```

Comment:
Why as `custom_objects_importer`? It's more simple to import it as `custom_objects` and do : 
```python
model = keras.models.load_model(args.model, custom_objects=custom_objects(args.backbone))
```